### PR TITLE
polished advanced t01

### PR DESCRIPTION
**Pull request by vbvg2008**

### CODE REVIEW

File: tutorial/advanced/t01_dataset.ipynb [hunk 1]
```diff
@@ -8,18 +8,26 @@
...
-    "    * Index split\n",
-    "* Global Dataset Editing\n",
-    "* BatchDataset\n",
-    "    * Deterministic batching\n",
-    "    * Distribution batching\n",
-    "    * Unpaired dataset\n",
+    "* [Dataset Summary](#ta01summary)\n",
+    "* [Dataset Splitting](#ta01splitting)\n",
+    "    * [Random Fraction Split](#ta01rfs)\n",
+    "    * [Random Count Split](#ta01rfs)\n",
```
Comment (TortoiseHam) [inline tutorial/advanced/t01_dataset.ipynb:14]: link duplicated

File: tutorial/advanced/t01_dataset.ipynb [hunk 2]
```diff
@@ -8,18 +8,26 @@
...
+    "    * [Index Split](#ta01is)\n",
+    "* [Global Dataset Editing](#ta01gde)\n",
+    "* [BatchDataset](#ta01bd)\n",
+    "    * [Deterministic Batching](#ta01deterministic)\n",
+    "    * [Distribution Batching](#ta01distribution)\n",
+    "    * [Unpaired Dataset](#Unpaird-Dataset)\n",
```
Comment (TortoiseHam) [inline tutorial/advanced/t01_dataset.ipynb:20]: link naming

File: tutorial/advanced/t01_dataset.ipynb [hunk 3]
```diff
@@ -252,14 +319,21 @@
...
+   "source": [
+    "<a id='ta01ud'></a>"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Unpaird dataset\n",
+    "### Unpaird Dataset\n",
```
Comment (TortoiseHam) [inline tutorial/advanced/t01_dataset.ipynb:336]: Unpaired

File: tutorial/advanced/t01_dataset.ipynb (hunk 1 above)
Comment (vbvg2008) [reply tutorial/advanced/t01_dataset.ipynb]: done

File: tutorial/advanced/t01_dataset.ipynb (hunk 2 above)
Comment (vbvg2008) [reply tutorial/advanced/t01_dataset.ipynb]: done

File: tutorial/advanced/t01_dataset.ipynb (hunk 3 above)
Comment (vbvg2008) [reply tutorial/advanced/t01_dataset.ipynb]: done

**Pull request merged**